core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/TestCASLockFactory.java

```diff
@@ -323,6 +325,12 @@ public Host getLocalHost()
                             {
                                 return null;
                             }
+
```

Comment:
This is overriding a method with the exact same content.

Reply:
No,its needed since this has no relation to the thing in AbstractCassandraTest.